functions/package.json

```diff
@@ -9,9 +9,8 @@
 		"build": "tsc",
 		"build:watch": "tsc --watch",
 		"serve": "npm run build:watch",
-		"serve:watch": "npm run build -- --watch & firebase emulators:start --project social-income-prod --config ../firebase.json --import ../seed",
-		"test": "firebase emulators:exec --only functions --only firestore --project social-income-prod --config ../firebase.json 'npm run test:app'",
-		"test:app": "jest --roots  src/"
+		"test": "firebase emulators:exec --only functions --only firestore --project social-income-prod --config ../firebase.json --import ../seed 'npm run test:app'",
```

Comment:
Why did you delete the serve:watch? I find it useful to work with.

Reply:
Oh sorry, this is a mistake. I thought that it can't be executed, which is totally wrong.